adoc/chapters/acknowledgements.adoc

```diff
@@ -48,8 +48,10 @@
   * Michael Wong, Codeplay
   * Peter Žužek, Codeplay
   * Matt Newport, EA
-  * Rasool Maghareh, Huawei Technologies Co. Ltd.
-  * Guansong Zhang, Huawei Technologies Co. Ltd.
+  * Rasool Maghareh, Huawei Technologies Co.
+    Ltd.
+  * Guansong Zhang, Huawei Technologies Co.
+    Ltd.
```

Comment:
Not sure we want these newlines, but it does not hurt.

Reply:
You can add more to the endAbbrev regexp in reflow.py.

Reply:
@oddhack would you prefer us to `fork` reflow.py or should we move `reflow` in a separate repo so it can be shared between projects?

Reply:
Go ahead and make a copy (you'll need a couple of other .py files from the scripts/ directory, see the import statements). I keep relevant scripts loosely synced up between the different repos using them but it's not worth the complexity of separate repos.